lightning/src/ln/channelmanager.rs

```diff
@@ -2947,6 +2981,11 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 						node_id: counterparty_node_id.clone(),
 						msg: announcement_sigs,
 					});
+				} else if chan.get().is_usable() {
+					channel_state.pending_msg_events.push(events::MessageSendEvent::SendChannelUpdate {
```

Comment:
is it appropriate to send this message if we haven't seen the locking block?

Reply:
Hmm, good point. The spec is mum on it (it only mentions that we have to wait until we *receive* a `funding_locked`) see https://github.com/lightningnetwork/lightning-rfc/blob/master/07-routing-gossip.md#requirements-3 so I think its fine. There isn't a ton of logical reason to wait, either, IMO, its just some fields the counterparty stores locally, so who cares when its received.

Reply:
OK, something to keep in mind if we see interop issues in this area.